include/RED4ext/Memory/Allocators.hpp

```diff
@@ -21,6 +21,7 @@ RED4EXT_ASSERT_SIZE(AllocationResult, 0x10);
 
 struct IAllocator
 {
+    virtual ~IAllocator() = default;
```

Comment:
Allocators don't have virtual destructors.

Reply:
For better understanding: each allocator is nothing but a unique VFT. They're never destructed and even copied by value. When another class holds allocator "reference", it doesn't use `IAllocator*` pointer, but instead copies the allocator instance (i.e. just copies VFT pointer).

Reply:
Yeah this change breaks the interface used by the engine, so it's expected to crash.